django/contrib/auth/backends.py

```diff
@@ -90,9 +96,15 @@ def has_module_perms(self, user_obj, app_label):
     def get_user(self, user_id):
         UserModel = get_user_model()
         try:
-            return UserModel._default_manager.get(pk=user_id)
+            user = UserModel._default_manager.get(pk=user_id)
         except UserModel.DoesNotExist:
             return None
+        if user:
+            is_active = getattr(user, 'is_active', None)
+            if (is_active or (is_active is False and self.allow_inactive_users) or
```

Comment:
What values would this condition have a different outcome than the current one?
`is_active or self.allow_inactive_users or is_active is None`